hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -4086,7 +4103,8 @@ public void copyLocalFileFromTo(File file, Path from, Path to) throws IOExceptio
             Progressable progress = null;
             PutObjectRequest.Builder putObjectRequestBuilder =
                 newPutObjectRequestBuilder(key, file.length(), false);
-            S3AFileSystem.this.invoker.retry("putObject(" + "" + ")", to.toString(), true,
+            final String d = to.toString();
```

Comment:
nit: Can have a better variable name

Reply:
done, reviewed that code and fixed the span logic for a single span across the entire process.